src/AI/CoreLogic.pm

```diff
@@ -1041,6 +1041,25 @@ sub processTransferItems {
 			redo;
 		}
 
+		#verify if we can carry the amount of item
+		#if you're sending to storage, doesn't need to check weight
+		#if there is no information about weight, there's no point in check
```

Comment:
It is compatible with item_weights it?

Reply:
actually, is only possible because of this implementation of allanon
i am just make good use of item_weights.txt :smile: 
i am using item_weights.txt, and if don't have the value, it will simply skip this piece of code and keep going